docs/content/use-cases/gitpod.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Usage with Gitpod"
+---
+
+## Gitpod
+
+[Gitpod](https://www.gitpod.io/) is an only development environment.
```

Comment:
Consider "Gitpod is a browser-based development environment."
or
"Gitpod is a browser only development environment."

But, it probably needs a bit more explanation.